docs/admin-api-clusters.md

```diff
@@ -9,37 +9,43 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 ````
 
-
 :::tip
 
- This page only shows **some frequently used operations**.
-
- - For the latest and complete information about `Pulsar admin`, including commands, flags, descriptions, and more, see [Pulsar admin docs](pathname:///reference/#/@pulsar:version_origin@/pulsar-admin/).
-
- - For the latest and complete information about `REST API`, including parameters, responses, samples, and more, see {@inject: rest:REST:/} API doc.
+This page only shows **some frequently used operations**. For the latest and complete information, see the **reference docs** below.
 
- - For the latest and complete information about `Java admin API`, including classes, methods, descriptions, and more, see [Java admin API doc](/api/admin/).
+ - [Pulsar CLI](reference-cli-tools.md) 
```

Comment:
Can we move this part out of the tip module and merge it into the following paragraph? The current structure looks a bit repetitive.

Reply:
@hangc0276 

This is the current doc:
<img width="1754" alt="image" src="https://user-images.githubusercontent.com/50226895/228117469-f1c56712-1c81-4a02-8b26-c8db102f5428.png">

Do you want me to consolidate it like this?
<img width="1495" alt="image" src="https://user-images.githubusercontent.com/50226895/228119785-61cdfed0-670f-43d4-810a-27ca183c0ba9.png">

Reply:
Moving the `Category` table out of the tip may be better.

Reply:
Make sense! Updated as below, PTAL.
<img width="1522" alt="image" src="https://user-images.githubusercontent.com/50226895/228125603-0e921a93-5a04-45aa-a932-e30921510fe6.png">